src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X/MvcRazorTemplateEngine.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+using Microsoft.AspNetCore.Razor.Language;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X
+{
+    /// <summary>
+    /// A <see cref="RazorTemplateEngine"/> for Mvc Razor views.
+    /// </summary>
+    public class MvcRazorTemplateEngine : RazorTemplateEngine
```

Comment:
hmmm. I'm not sure that we need this. Is this used anywhere?

Reply:
Pretty sure we can nuke